### DISCUSSION TITLE
Is there any paid support for Quartz?

### DISCUSSION THREAD

**Discussion by viveknuna** [quartznet/quartznet#2167] [Q&A]
We are using quartz in our service, but there are issue related to high memory usage by the service which is using Quartz, now it can run only on one instance/pod and many schedules being into error state or misfires.

I have posted many questions on this repo, but I haven’t got any support here, so there any expert/ freelancer who could help me on these things in my project on paid basis, I’m willing to pay money as well.

please recommend.

**Comment by lahma** [comment]
I'm sorry to hear that you have had such an unpleasant journey with Quartz.NET, but as you know this is a free, open-source project, which does not offer such services like consulting at the moment. I personally just don't have the capacity to diagnose problems and find the eventual problem.

Quartz.NET is being used by a large community/corporations and without such high memory usage problems so I generally would suggest that the problem is on your side, not Quartz's, how bad that might feel.

I hope that you would be able to find the problem by using profiling tools against your codebase. I would also be very happy to see our community to step up and help with the questions that you've raised. OSS project basically needs to have a community to tackle such problems.

The problems you've posted might not be trivial to diagnose, but as you've got the source code available you could debug and decipher some behavior to find the eventual underlying problem.

Anyone from the community, please raise your hand here too if you would like to diagnose such problems and be compensated from your work,.

**Reply by viveknuna** [reply]
Thank your @lahma for your inputs. Yes I also suspect that the problem is at our end. That's why I need someone's help to identify what are we doing which is causing all these issues.

**Reply by xmateosx** [reply]
I'm having problems with IIS Integration and am looking for help, for which I am also willing to pay for.

**Comment by Sicos1977** [comment]
If you have a memory leak somewhere you could try to use a tool like Resharper dotTrace profiler. I used it in the past to find out why I had a simular problem (not Quartz.Net related). It is a very nice tool to find out why something is eating up al the memory.

https://www.jetbrains.com/profiler/

There is a 30 day trail version.
